aten/src/ATen/native/quantized/cudnn/Conv.cpp

```diff
@@ -201,7 +212,7 @@ void raw_cudnn_convolution_forward_out(
   auto run = [&](cudnn_frontend::ManagedOpaqueDescriptor plan_desc) {
     auto workspace_size = 0;
     auto workspace = at::empty({workspace_size}, input.options().dtype(kByte));
-    void *data_ptrs[] = {reinterpret_cast<int8_t*>(input.data_ptr()), conv_output.data_ptr(), reinterpret_cast<int8_t*>(weight.data_ptr()), requantize_multiplier_tensor.data_ptr(), output.data_ptr()};
+    void *data_ptrs[] = {reinterpret_cast<int8_t*>(input.data_ptr()), conv_output.data_ptr(), reinterpret_cast<int8_t*>(weight.data_ptr()), requantize_multiplier_tensor.data_ptr(), quantized_output.data_ptr()};
```

Comment:
we can do a reinterpret_cast for quantized_output.data_ptr() here as well I think:
```
reinterpret_cast<int8_t*>(quantized_output.data_ptr())
```
original data type should be `qint8_t*` but we can cast it to `int8_t*`